apps/settings/locales/settings.en-US.properties

```diff
@@ -1301,6 +1301,7 @@ improveBrowserOSTitle=Improve {{brandShortName}}
 sendMozillaFeedback=Send Mozilla Feedback
 performanceAndUsageData=Performance and Usage Data
 sharePerformanceAndUsageData=Submit data
+openAppStats=Open App Statistics
```

Comment:
@andymckay I liked the suggestion on the bug 'App usage by region', I have that in my latest that I'm about to push.. wdyt?